rqt_bag/src/rqt_bag/bag_widget.py

```diff
@@ -335,6 +336,15 @@ def load_bag(self, filename):
         # self clear loading filename
 
     def _handle_save_clicked(self):
+        # If we are currently recording, close the bag.
+        if self._recording:
+            self._timeline.stop_recorder()
+            self._recording = False
+            self.save_button.setEnabled(False)
+            self.load_button.setEnabled(True)
+            return
+        # todo: Enable this code pathway when support for extracting regions
```

Comment:
Our typical style here is `TODO(username)` following the Google guidelines:

```suggestion
        # TODO(Yadunund): Enable this code pathway when support for extracting regions
```